File: C/Easy/2678.c
int countSeniors(char **details, int detailsSize)
{
    int count = 0;
    char *detail;
    for (int i = 0; i < detailsSize; i++)
    {
        detail = details[i];
        if (detail[11] > '6' || (detail[11] == '6' && detail[12] > '0'))
        {
            count++;
        }
    }

    return count;
}

File: Java/Easy/2678.java
class Solution {
    public int countSeniors(String[] details) {
        int count = 0;

        for (String string : details) {
            if (string.charAt(11) > '6' || (string.charAt(11) == '6' && string.charAt(12) > '0')) {
                count++;
            }
        }

        return count;
    }
}